kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/JbpmKieServerExtension.java

```diff
@@ -474,6 +478,41 @@ protected void addAsyncHandler(KModuleDeploymentUnit unit, InternalKieContainer
         }
     }
 
+    protected void addTaskBAMEventListener(final KModuleDeploymentUnit unit, final InternalKieContainer kieContainer) {
+        final DeploymentDescriptor descriptor = getDeploymentDescriptor(unit, kieContainer);
+        descriptor.getBuilder().addTaskEventListener(
+                new NamedObjectModel(
+                        "mvel",
+                        "taskBAM",
+                        "new org.jbpm.services.task.lifecycle.listeners.BAMTaskEventListener(" +
+                            "org.jbpm.runtime.manager.impl.jpa.EntityManagerFactoryManager.get().getOrCreate(\"" + persistenceUnitName + "\")" +
```

Comment:
Bam task listener should be without entity manager factory https://github.com/droolsjbpm/jbpm/blob/master/jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/lifecycle/listeners/BAMTaskEventListener.java#L71 so it will use one taken from taskcontext, see here https://github.com/droolsjbpm/jbpm/blob/master/jbpm-human-task/jbpm-human-task-jpa/src/main/java/org/jbpm/services/task/persistence/PersistableEventListener.java#L44